src/course-home/data/api.js

```diff
@@ -137,6 +137,10 @@ export function normalizeOutlineBlocks(courseId, blocks) {
           resumeBlock: block.resume_block,
           sequenceIds: block.children || [],
           hideFromTOC: block.hide_from_toc,
+          completionStat: {
+            completed: block.completion_stat?.completion,
+            total: block.completion_stat?.completable_children,
+          },
```

Comment:
I don't think this is currently data that is returned by the backend. I would either:
1. Try to get the information from somewhere else we already have access to that information.
2. Start a discussion to update the existing contract and scope that new backend work.